tools/ffi_coverage/src/allowlist.rs

```diff
@@ -94,9 +94,13 @@ lazy_static::lazy_static! {
     // Ignore if this is a substring of any path
     // keep this small
     pub static ref IGNORED_SUBSTRINGS: &'static [&'static str] = &[
-        // _unstable constructors cover these
-        "with_any_provider",
-        "with_buffer_provider",
+        // compiled data constructors cover these
+        "_with_any_provider",
+        "_with_buffer_provider",
+        "_unstable",
+
+        // compiled data constructors cover these
+        "load_",
```

Comment:
Nit: I'm a little hesitant to do a blanket suppression of `load_` because that seems like a common enough prefix that it could be used in function names that aren't about providers. I think I favor `rust_link`ing these (a little annoying but the scope of the problem is limited)